.ci.yaml

```diff
@@ -2251,6 +2261,18 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: android_lifecycles_test
 
+  - name: Linux_samsung_a02 openpay_benchmarks__scroll_perf
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      openpay: "true"
+      tags: >
+        ["devicelab", "android", "linux", "openpay", "samsung", "a02"]
+      task_name: openpay_benchmarks__scroll_perf
+    scheduler: luci
```

Comment:
Oops, sorry I missed this line. Now everything is via cocoon scheduling, could you remove this line?

Reply:
https://github.com/flutter/flutter/pull/108126